content/pages/code-of-conduct/index.de.md

```diff
@@ -0,0 +1,103 @@
+
+Konkret heißt das für uns, dass wir
+ * ... uns gegenseitig ausreden lassen und uns einander zuhören.
+ * ... die Meinung anderer respektieren, auch wenn sie von der eigenen abweicht. Dies gilt, solange Meinungen nicht im Konflikt mit unseren Zielen und Grundwerten stehen. Mehr zu unseren Grundwerten folgt im Detail weiter unten.
+ * ... wir einander bereitwillig helfen.
+ * ... wir auf unsere Mitmenschen und unsere Umgebung achten. Das heißt beispielsweise, dass wir hinter uns aufräumen und genutzte Orte sauber hinterlassen, dass wir nicht unnötig laut oder anderweitig störend sind, und dass wir nachhaltig mit unseren Ressourcen umgehen.
+
+Im temporärhaus kommen verschiedene Menschen mit vielen unterschiedlichen Fähigkeiten, Kenntnissen und Hintergründen zusammen. Wir verstehen das als große Chance, denn aus Diversität heraus können wunderbare Dinge entstehen. Eine Voraussetzung dafür ist, dass wir alle Personen unabhängig von ihrem Interessensspektrum so respektieren wie sie sind und wir alle Fähigkeiten und Kenntnisse als gleich wertvoll erachten.
+Bitte achte darauf, individuelle Stärken oder Schwächen, technische Ausstattung oder persönliche Vorlieben nicht herablassend oder belästigend zu kommentieren, sondern versuche eher interessiert von anderen zu lernen und neue Dinge auszuprobieren.
+
+Wichtig ist uns hierbei, Bezug zu nehmen auf die [Hacker\*innen-Ethik](https://www.youtube.com/watch?v=gP2wrzQDS9w), die wir in unseren Räumen und Veranstaltungen aktiv fördern. Falls du die Hacker\*innen-Ethik noch nicht kennen solltest, kannst du dir gerne das oben verlinkte Video dazu ansehen, oder [beim Chaos Computer Club eine textuelle Version](https://www.ccc.de/hackerethik) nachlesen.
+
+
+## Unsere Grundwerte und allgemeine Regeln im gemeinsamen Umgang
+Neben den bisher genannten, eher allgemeinen Umgangsformen, gibt es einige weitere Verhaltensweisen, die wir in unserer Gemeinschaft und in unseren Räumlichkeiten als selbstverständlich ansehen:
+
+* Wir respektieren, wie unser Gegenüber angesprochen werden möchte. Das bezieht sich nicht nur auf den Namen, sondern auch auf die Pronomen (z.B. sie, es, xie, they oder er). Bist du dir unsicher, mit welchen Pronomen eine Person angesprochen werden möchte, kannst du sie einfach fragen oder gegenderte Sprache vermeiden, indem du den Namen der Person verwendest.
+* Sei integrativ und unterstützend gegenüber Personen, die weniger selbstbewusst sind oder andere, bzw. verschiedene Arten der Kommunikation nutzen. Dadurch erhalten alle die Gelegenheit, sich einzubringen.
+* Kommentare über Lebensstil und Angewohnheiten anderer Personen, einschließlich Ernährung, Gesundheit, Elternschaft, Drogen und Beschäftigungen sind unerwünscht und sollten vermieden werden.
```

Comment:
Wenn man diesen Punkt wörtlich umsetzt, dürfte man nicht kommentieren, dass ich mir angewöhnt habe eine Kutte :metal: im Alltag zu tragen, auch wenn ich es nicht unerwünscht fände, wenn jemand über meine Eigenheiten auf Augenhöhe spricht.

Im Gegenteil eher fände ich zu viel Distanz eine gegenläufige Kraft zum Gedanken der Annäherung diverser Menschen.

_Vorschlag_

> **Neugierige** Kommentare über Lebensstil und Angewohnheiten anderer Personen, einschließlich […] sind ~~unerwünscht~~ **oft unsensibel** und sollten **eher** vermieden werden.

Mit dem weicheren wording geht der Zeiger mehr auf "der Ton macht die Musik", was glaube ich häufig die Empfindung immens beeinflusst

Reply:
Das so als Inspiration von mir. Ansonsten schöne Gliederung und approved :heavy_check_mark:

Reply:
Hab da bisschen drüber nachgedacht. Ist ein guter Einwand, aber so wie es aktuell formuliert ist (mit den Beispielen Ernährung, Gesundheit, ...) ist es aus dem Kontext heraus hoffentlich schon verständlich, dass man einfach nur nicht private Sitten von Leuten thematisieren soll.
Dein Beispiel, also das Tragen einer Kutte, fällt da zwar eventuell drunter, aber andererseits könnte es ja auch sein, dass jemand bei uns ankommt der wegen der eigenen Kleidung hart in der Schule gemobbt wird und sich sofort unwohl/unsicher/(sogar bedroht?) fühlt wenn Kommentare über Kleidung gemacht werden.
Fände es da wichtiger, solche Personen zu schützen, und in deinem Fall durch deine offene Art zu versuchen, Leute zu Gesprächen über deine Kleidung zu animieren weil es bei dir explizit gewünscht ist.
Quasi safe baseline, Ausnahmen pro Person wenn gewünscht.